modules/godot/databags/databag_timer.h

```diff
@@ -55,11 +56,29 @@ class TimersDatabag : public godex::Databag {
 		timers[p_timer_handle.timer_index].end_time = p_microseconds;
 	}
 
+	// For internal usage only: has no safety checks.
+	_FORCE_INLINE_ uint64_t internal_get_now() const {
+		return now - paused_time;
+	}
+
 public:
 	TimersDatabag() {}
 
-	/// Used by `WorldECS` to update time.
-	void set_now(const uint64_t p_now);
+	/// Used by `Utilities Bundle` to update time by setting a new now.
```

Comment:
I think better to mention the System name, since the user may decide to just use the system and not the entire bundle.